src/xdocs/config_annotation.xml

```diff
@@ -1355,15 +1355,19 @@ package com.example.annotations.packageannotation; //ok
       </subsection>
 
       <subsection name="Examples" id="SuppressWarningsHolder_Examples">
```

Comment:
Please add example for property of Holder.
We need to show how configure it and how it works

Reply:
@romani  I think  example for ```aliasList``` is already added in documentation and I have created config also,  for ```ParameterNumberCheck``` which is written as ```paramnum``` 
![Screenshot from 2022-05-08 09-44-37](https://user-images.githubusercontent.com/97679350/167281433-909962df-f637-490a-92d7-c06fc1acf9c7.png)

Reply:
Please share CLI out out with aliasList in config.
`aliasList` must be explicitly set in config file.

Reply:
Users should be copy pasting config and javacode to their file, run CLI as you do at https://github.com/checkstyle/checkstyle/pull/11542#issuecomment-1120175272 and reproduce behavior

Reply:
@romani can you please guide on to set which value is to be used overhere
```
<module name="SuppressWarningsHolder">
    <property name="aliasList" value=" " />
</module>
```

Reply:
@romani, ping

Reply:
https://checkstyle.org/config_annotation.html#SuppressWarningsHolder_Properties

>Specify aliases for check names that can be used in code within SuppressWarnings. | String[] in a format of comma separated attribute=value entries. The attribute is the fully qualified name of the Check and value is its alias.

And find in text of our doc
>If aliasList property was provided you can use your own names e.g below code will work if there was provided a ParameterNumberCheck=paramnum in the aliasList:

Reply:
done